README.md

```diff
@@ -110,6 +110,15 @@ uom = {
  * `use_serde` -- Feature to enable support for serialization and deserialization of quantities
    with the [Serde][serde] crate. Disabled by default.
 
+Note that you need to manually enable Serde support for the optional `num-rational` and
+`num-bigint` dependencies if you intend to (de)serialize quantities based on the related storage types.
+To do so, you can add one or both of the following lines to your Cargo.toml:
+
+```toml
+num-rational = { version = "*", features = ["serde"] }
+num-bigint = { version = "*", features = ["serde"] }
+```
+
```

Comment:
Sorry I lost track of this PR. I had a lot going on at the end of September and just didn't follow up once things calmed down.

 * Include the additional text inside the `use_serde` bullet point.
 * Wrap at the 100 character mark.
 * I've included some updated text below that calls out this is only required for certain underlying storage types.
 * Once text is finalized it also needs to be added to `lib.rs` https://github.com/iliekturtles/uom/blob/fe7f0b38ca1daffe67dd60b3adb1676766f0caec/src/lib.rs#L98-L102

```suggestion
   [Serde][serde] support for the `big*` and `rational*` underlying storage types requires manually
   enabling the `serde` feature for the `num-rational` and `num-bigint` crates. To do so, you can
   add one or both of the following lines to your `Cargo.toml`:

   ```toml
   num-rational = { version = "*", features = ["serde"] }
   num-bigint = { version = "*", features = ["serde"] }
   ```

```